com.atlauncher.ATLauncher.metainfo.xml

```diff
@@ -0,0 +1,103 @@
+            <li>Allows quick and easy install of modpacks, no more messing around trying to install mods</li>
+            <li>A range of different packs, and more being added all the time</li>
+            <li>Multiple instances. Install a pack as many times as you like without messing up a previous install</li>
+            <li>Multiple accounts allow you to login as different users from the one install</li>
+            <li>Private instances. This allows you to only install instances under your account so people can't mess up your worlds</li>
+            <li>1 click backup of your saves</li>
+            <li>Easy server creation from within the Launcher</li>
+            <li>Install packs from CurseForge and FTB</li>
+            <li>Mod management for instances, including auto download and installation of mods, maps and resource packs from CurseForge and Modrinth</li>
+            <li>Support for both Mojang and Microsoft accounts</li>
+            <li>More features always being added</li>
+        </ul>
+    </description>
+    <categories>
+        <category>Game</category>
+    </categories>
+    <url type="homepage">https://atlauncher.com/</url>
+    <url type="bugtracker">https://github.com/ATLauncher/ATLauncher/issues</url>
+    <url type="faq">https://atlauncher.com/help</url>
+    <url type="help">https://atl.pw/discord</url>
+    <url type="translate">https://translate.atlauncher.com/</url>
+    <url type="contact">https://atlauncher.com/contact-us</url>
+    <launchable type="desktop-id">com.atlauncher.ATLauncher.desktop</launchable>
+    <project_license>GPL-3.0</project_license>
+    <developer_name>The ATLauncher Team</developer_name>
+    <screenshots>
+        <screenshot type="default">
+            <image type="source" width="1186" height="693">https://atlauncher.com/assets/images/features/home.png</image>
+        </screenshot>
+        <screenshot>
+            <image type="source" width="636" height="393">https://atlauncher.com/assets/images/features/console.png</image>
+        </screenshot>
+        <screenshot>
+            <image type="source" width="1186" height="693">https://atlauncher.com/assets/images/features/packs.png</image>
+        </screenshot>
+        <screenshot>
+            <image type="source" width="1186" height="693">https://atlauncher.com/assets/images/features/instances.png</image>
+        </screenshot>
+        <screenshot>
+            <image type="source" width="1186" height="693">https://atlauncher.com/assets/images/features/servers.png</image>
+        </screenshot>
+        <screenshot>
+            <image type="source" width="1186" height="693">https://atlauncher.com/assets/images/features/add-mods.png</image>
+        </screenshot>
+        <screenshot>
+            <image type="source" width="1186" height="693">https://atlauncher.com/assets/images/features/settings.png</image>
+        </screenshot>
+    </screenshots>
+    <update_contact>ryan.dowling_AT_atlauncher.com</update_contact>
```

Comment:
Will it be maintained officially by the upstream developer?

Reply:
Not as it stands